sirepo/job.py

```diff
@@ -40,6 +40,9 @@
 #: path supervisor registers to receive requests from job_process for file PUTs
 DATA_FILE_URI = '/job-cmd-data-file'
 
+#: path supervisor registers to receive requests from job_process for file GETs/PUTs
+DB_FILE_URI = '/db-file'
```

Comment:
@robnagler do you have a better name than db_file?

Reply:
maybe /sim-db-file.